compiler/rustc_mir_build/src/build/matches/test.rs

```diff
@@ -234,7 +234,7 @@ impl<'a, 'tcx> Builder<'a, 'tcx> {
                     };
                     TerminatorKind::if_(self.tcx, Operand::Copy(place), true_bb, false_bb)
                 } else {
-                    // The switch may be inexhaustive so we have a catch all block
+                    // The switch may be inexhaustible so we have a catch all block
```

Comment:
We use exhaustive as name inside code elsewhere.

Reply:
"exhaustive" is also a technical term and the right one to use here, and "inexhaustive" is its correct negation.